mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -1162,15 +1162,25 @@ def execute_block_function(
             # Initialize module
             if self.language == BlockLanguage.PYTHON:
                 try:
+                    block_uuid = self.uuid
+                    block_file_path = self.file_path
+                    if self.replicated_block:
+                        block_uuid = self.replicated_block
+                        block_file_path = Block(
```

Comment:
Could you add a property method for replicated_block_object? I saw this code used multiple times in this file.

Reply:
sure